sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/implementation/converters/PlayAudioConverter.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callingserver.implementation.converters;
+
+import com.azure.communication.callingserver.implementation.models.PlayAudioRequest;
+import com.azure.communication.callingserver.models.PlayAudioOptions;
+
+public class PlayAudioConverter {
+    /**
+     * Convert PlayAudioOptions into PlayAudioRequest
+     * @param audioFileUri Audio file uri to be added during conversion
+     * @param playAudioOptions PlayAudioOptions to be converted
+     * @return PlayAudioRequest
+     */
+    public static PlayAudioRequest convert(String audioFileUri, PlayAudioOptions playAudioOptions) {
+        PlayAudioRequest playAudioRequest = new PlayAudioRequest();
```

Comment:
You should have a check here that the playAudioOptions is not null. I dont see this check where convert() is being called.

Reply:
null check is added